Source/Core/VideoBackends/OGL/Render.cpp

```diff
@@ -711,10 +712,13 @@ Renderer::Renderer()
   // options while running
   g_Config.bRunning = true;
 
-  glStencilFunc(GL_ALWAYS, 0, 0);
-  glBlendFunc(GL_ONE, GL_ONE);
+  // The stencil is used for bounding box emulation when SSBOs are not available
+  glDisable(GL_STENCIL_TEST);
+  glStencilFunc(GL_ALWAYS, 1, 0);
+  glStencilOp(GL_KEEP, GL_INCR, GL_INCR);
```

Comment:
I was thinking that INCR might be better because it doesn't require draws to be performed in a specific order, but I can change it.

Yeah, I have no clue about the second argument either. I've tried Paper Mario: TTYD with both possibilities, and it doesn't seem to matter.